docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+
+  descartes_rpa:
+    build:
+      context: .
+    volumes:
+      - /home/joao/gsoc/descartes-rpa/data/output:/descartes_rpa/data/output
```

Comment:
pass in users path as an environment variable for compatibility

Reply:
Now changed fullpath to relative path to data folder